docs/quickstart-docker.md

```diff
@@ -0,0 +1,141 @@
+
+    | Option | Description |
+    |---|---|
+    | `--name`  |  Provides a name to the container. If you do not use this option, Docker adds a random name.  |
+    | `--volumes-from` | Refers to Percona Server for MySQL and indicates that you intend to use the same data as the `psmysql` container.|
+    | `it` | Interacts with the container and be a pseudo-terminal. |
+    | `--user root` | Sets the user to root inside the container. This option is required to access the MySQL data directory and run the xtrabackup command. |
+    |`backup_8034` |Indicates the directory inside the container where the backup files are stored. |
+    | `-v` | Mounts a volume from the host machine or another container to the container that is being run. |
+    | `backupvol` | Indicates the persistent storage for the database. |
+    | `psmysql`    | Indicates the Percona Server for MySQL container.  |
+    |`--password`   | Prompts the user to enter the password for the root user. For convenience you can add the password for the root user to this option, for example, `--password=secret`. Then the password will be passed automaticaly when running the command. Note that if you specify the password with `--password=secret`, the password is visible in `docker ps`, `docker ps -a` (docker history) and regular ps command.          |
+    | `8.0.34` | Use this tag to specify a specific version. |
+
+    When running Percona XtraBackup from a container and connecting to a Percona Server container, we recommend using the `–-user root` option in the Docker command.
+
+    For this document, we are using the 8.0.34 tag. In Docker, a tag is a label assigned to an image. Tags are used to maintain different versions of an image. If we did not add a tag, Docker uses latest as the default tag and downloads the latest image from [percona/percona-xtrabackup on the Docker Hub](https://hub.docker.com/r/percona/percona-xtrabackup).
+
+    * Run a Docker container example
+
+        ```{.bash data-prompt="$"}
+        $ docker run --name pxb --volumes-from psmysql -v backupvol:/backup_8034 -it --user root percona/percona-xtrabackup:8.0.34 /bin/bash -c "xtrabackup --backup --datadir=/var/lib/mysql/ --target-dir=/backup_8034 --user=root --password; xtrabackup --prepare --target-dir=/backup_8034"
+        ```
+
+        You are prompted to enter the password, which is `secret`, in our example.
+
+        ```{.text .no-copy}
+        2023-12-21T16:16:10.796716-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized server arguments: --datadir=/var/lib/mysql/2023-12-21T16:16:10.799979-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized client arguments: --backup=1 --target-dir=/backup_8034 --user=root --password
+        
+        Enter password:
+        ```
+    
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+
+        2023-12-14T15:45:16.277503-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized server arguments: --datadir=/var/lib/mysql/ 
```

Comment:
we may be able to see the first and last section and use an ellipses instead of the entire backup

Reply:
Added